schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+id BIGSERIAL NOT NULL,
```

Comment:
[OPTIONAL]
- I think it would be a good idea to mark this id as a primary key. For example `PRIMARY KEY (id)`